src/components/instance-configurator/InstanceConfigurator.jsx

```diff
@@ -79,24 +80,31 @@ class InstanceConfigurator extends React.Component {
         },
     ];
 
+    backHome = () => {
+        this.props.history.push("/");
+    };
+
     render() {
         const { d2 } = this.props;
 
         return (
-            <div>
-                <ObjectsTable
-                    key={this.state.tableKey}
-                    d2={d2}
-                    model={d2.models.dataSet}
-                    columns={this.columns}
-                    detailsFields={this.detailsFields}
-                    pageSize={10}
-                    initialSorting={this.initialSorting}
-                    actions={this.actions}
-                    onCreate={this.onCreate}
-                    list={Instance.list}
-                />
-            </div>
+            <React.Fragment>
+                <PageHeader title={i18n.t("Instances")} onBackClick={this.backHome} />
+                <div style={{ marginTop: -10 }}>
```

Comment:
As @SferaDev already noted, let's use `withStyle`

Reply:
I left this on hold while waiting for the ObjectsTable comment to be answered, I'll change it and we'll remove it when we update d2-ui-components.